client/packages/system/src/IndicatorsDemographics/DetailView/IndicatorsDemographics.tsx

```diff
@@ -46,14 +46,25 @@ const IndicatorsDemographicsComponent = () => {
   const t = useTranslation();
 
   const { draft, setDraft } = useDemographicData.indicator.list(headerDraft);
+  const baseYear = headerDraft?.baseYear ?? 2024;
```

Comment:
Do you think we should be defaulting this to something like `new Date().getFullYear()` ?

Reply:
yes, much better - thanks